orion/primitives/timeseries_errors.py

```diff
@@ -26,6 +27,9 @@ def regression_errors(y, y_hat, smoothing_window=0.01, smooth=True):
         smooth (bool):
             Optional. Indicates whether the returned errors should be smoothed with EWMA.
             If not given, `True` is used.
+        mask (bool):
+            Optional. Mask the start of anomaly scores.
+            If not given, `False` is used.
 
```

Comment:
thank you @dyuliu for making the changes.

can you please add the docstrings of `masking_window`?

Reply:
good catch. Done.